support-frontend/assets/pages/digital-subscription-checkout/digitalSubscriptionCheckout.jsx

```diff
@@ -54,12 +60,14 @@ const content = (
   <Provider store={store}>
     <Page
       header={<HeaderWrapper />}
-      footer={<DigitalFooter />}
+      footer={<DigitalFooter orderIsAGift={orderIsAGift} />}
     >
       <CheckoutStage
-        checkoutForm={<CheckoutForm />}
-        thankYouContentPending={<ThankYouPendingContent includePaymentCopy {...thankyouProps} />}
-        thankYouContent={<ThankYouContent {...thankyouProps} />}
+        checkoutForm={orderIsAGift ? <CheckoutFormGift /> : <CheckoutForm />}
```

Comment:
What do you think about getting rid of all of these ternaries and moving the logic up a level so this whole block would change to:
```javascript
const content = orderIsAGift ? (
  <Provider store={store}>
    <Page
      header={<HeaderWrapper />}
      footer={<DigitalFooter orderIsAGift />}
    >
      <CheckoutStage
        checkoutForm={<CheckoutFormGift />}
        thankYouContentPending={<ThankYouGift { ...thankyouProps} pending />}
        thankYouContent={<ThankYouGift {...thankyouProps} />}
        subscriptionProduct="DigitalPack"
      />
    </Page>
  </Provider>
) : (
  <Provider store={store}>
    <Page
      header={<HeaderWrapper />}
      footer={<DigitalFooter orderIsAGift={false} />}
    >
      <CheckoutStage
        checkoutForm={<CheckoutForm />}
        thankYouContentPending={<ThankYouPendingContent includePaymentCopy {...thankyouProps} />}
        thankYouContent={<ThankYouContent {...thankyouProps} />}
        subscriptionProduct="DigitalPack"
      />
    </Page>
  </Provider>
);
```

Reply:
Yes, it had got a little messy. I will make that change.